src/cc/export/helpers.h

```diff
@@ -90,6 +90,14 @@ struct _name##_table_t { \
 __attribute__((section("maps/perf_array"))) \
 struct _name##_table_t _name
 
+// Identifier for current CPU used in perf_submit and perf_read
+// Prefer use BPF_F_CURRENT_CPU flag, falls back to call helper for older kernel
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 8, 0)
```

Comment:
not quite a 'forward port', but there are cases where features available in the older kernels will be disabled in the newer ones via .config